src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -158,7 +164,7 @@ public async Task<ActionResult> GetMessageBoxInstance(int instanceOwnerPartyId,
             [FromRoute] Guid instanceGuid)
         {
             string instanceId = $"{instanceOwnerPartyId}/{instanceGuid}";
-            string[] eventTypes = new string[]
+            string[] eventTypes =
             {
                 InstanceEventType.Created.ToString(),
                 InstanceEventType.Deleted.ToString(),
```

Comment:
Not possible to att the suggestion at the end, but that's probably where it should be placed

```suggestion
                InstanceEventType.Deleted.ToString(),
                InstanceEventType.SubstatusUpdated.ToString(),
```

Reply:
Added the change from your previous PR for sub status.